lib/browser/api/auto-updater/squirrel-update-win.ts

```diff
@@ -79,20 +77,25 @@ const spawnUpdate = function (args, detached, callback) {
 }
 
 // Start an instance of the installed app.
-exports.processStart = function () {
+export const processStart = () => {
   return spawnUpdate(['--processStartAndWait', exeName], true, function () {})
 }
 
+type SquirrelUpdateInfo = {
```

Comment:
We could move that to `Electron.*`, but I think it's fine here, too

Reply:
If it was re-used for the macOS impl or something I totally would extract it higher, but this purely lives for this single fn 🤔